src/helpers/geoChart/viewport.ts

```diff
@@ -0,0 +1,89 @@
+// (C) 2020 GoodData Corporation
+import get = require("lodash/get");
+import mapboxgl = require("mapbox-gl");
+
+import { DEFAULT_CENTER, DEFAULT_ZOOM, VIEWPORTS } from "../../constants/geoChart";
+import { IGeoConfig, IGeoConfigViewport, IGeoLngLat, IGeoLngLatBounds } from "../../interfaces/GeoChart";
+
+interface IGeoViewport {
+    bounds?: mapboxgl.LngLatBoundsLike;
+    center?: IGeoLngLat;
+    zoom?: number;
+}
+
+export function getViewportOptions(data: IGeoLngLat[], config: IGeoConfig): IGeoViewport {
+    const center: IGeoLngLat = get<IGeoConfig, "center">(config, "center");
+    const zoom: number = get<IGeoConfig, "zoom", number>(config, "zoom", DEFAULT_ZOOM);
+    const { area }: IGeoConfigViewport = get<IGeoConfig, "viewport", {}>(config, "viewport", {});
```

Comment:
Default value is empty object, is it correct? I see we define type of `VIEWPORTS` is `{ [key: string]: mapboxgl.LngLatBoundsLike }`, so VIEWPORTS[{}] still `return undefined`